pcdsdevices/spectrometer.py

```diff
@@ -318,3 +318,29 @@ class TMOSpectrometer(BaseInterface, GroupDevice):
     yag_y = Cpt(BeckhoffAxisNoOffset, ':MMS:07', kind='normal')
     yag_z = Cpt(BeckhoffAxisNoOffset, ':MMS:08', kind='normal')
     yag_theta = Cpt(BeckhoffAxisNoOffset, ':MMS:09', kind='normal')
+
+
+class HXRSpectrometer(BaseInterface, GroupDevice):
+    """
+    HXR Single Shot Spectrometer motion components class.
+
+    Parameters:
+    -----------
+    prefix : str
+        Motors base PV
+
+    name : str
+        Alias for the device
+    """
+
+    tab_component_names = True
+
+    # Motor components: can read/write positions
+
+    krst = Cpt(IMS, ':441:MOTR', kind='normal')  # crystal y
+    ksta = Cpt(IMS, ':442:MOTR', kind='normal')  # crystal angle
+    cama = Cpt(IMS, ':443:MOTR', kind='normal')  # camera angle
+    camd = Cpt(IMS, ':444:MOTR', kind='normal')  # camera distance
+    camy = Cpt(IMS, ':447:MOTR', kind='normal')  # camera y
+    cami = Cpt(IMS, ':445:MOTR', kind='normal')  # camera iris
+    fw = Cpt(IMS, ':446:MOTR', kind='normal')  # filter wheel, tbd if necessary
```

Comment:
The component's doc field might be a better spot for the comments

Reply:
Is Sanghoon happy with these names?

Reply:
These names follow what is currently in the hxs ioc.